tests/bluetooth/bsim_bt/compile.source

```diff
@@ -0,0 +1,36 @@
+function compile(){
```

Comment:
Remember to add the copyright + license message so compliance passes

Reply:
Thanks